PYME/recipes/localisations.py

```diff
@@ -426,6 +426,25 @@ def execute(self, namespace):
     def hide_in_overview(self):
         return ['columns']
 
+    @property
+    def default_view(self):
+        import wx
```

Comment:
What is this doing here?

Reply:
IE we shouldn't need to test for wx.App

Reply:
Can delete. Was following the example you told me to use as a template, `recipes.tablefilters`, line 71.

Reply:
sorry, the ExtractTableChannel one would probably have been a better example. Have just made a few tweaks (and about to make a couple more).